install.sh

```diff
@@ -191,28 +204,59 @@ done
 
 ## workaround https://github.com/skycocker/chromebrew/issues/3305
 sudo ldconfig &> /dev/null || true
-
-# create symlink to 'crew' in ${CREW_PREFIX}/bin/
+echo -e "\n${YELLOW}Creating symlink to 'crew' in ${CREW_PREFIX}/bin/${RESET}"
+echo -e "${GRAY}"
 ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
+echo -e "${RESET}"
+
+echo -e "${YELLOW}Setup and synchronize local package repo...${RESET}"
+echo -e "${GRAY}"
+
+# Remove old git config directories if they exist
+rm -rf "${CREW_LIB_PATH}"/{.git,.crewsetuptmp}
+
+# Do a minimal clone, which also sets origin to the master/main branch
+# by default. For more on why this setup might be useful see:
+# https://github.blog/2020-01-17-bring-your-monorepo-down-to-size-with-sparse-checkout/
+git clone --depth=1 --filter=blob:none --no-checkout "https://github.com/${OWNER}/${REPO}.git" "${CREW_LIB_PATH}"/.crewsetuptmp
 
-# prepare sparse checkout .rb packages directory and do it
+# move .git directory to final location and remove tmp folder.
+mv "${CREW_LIB_PATH}"/.crewsetuptmp/.git "${CREW_LIB_PATH}"/ && rm -rf "${CREW_LIB_PATH}"/.crewsetuptmp
```

Comment:
I think we're spending too much effort to support existing installs.  It's SOOO much easier just to blow away the old and start fresh rather than working around already installed files which are most likely corrupt anyway if another install is attempted.